apps/files/src/components/FilesAppBar.vue

```diff
@@ -122,7 +122,8 @@ export default {
       return this.$route.params.item === undefined ? this.configuration.rootFolder : this.$route.params.item
     },
     url () {
-      const path = this.item === '' ? (this.configuration.rootFolder ? `${this.configuration.rootFolder}/` : '/') : `${this.item}/`
+      let path = this.item === '' ? (this.configuration.rootFolder ? `${this.configuration.rootFolder}` : '') : `${this.item}`
```

Comment:
please double check the default value of rootFolder - I think it is / - no idea if this logic is still needed

Reply:
alright

Reply:
done